### ISSUE TITLE
Feature Request: Disable Config Option

### ISSUE THREAD

**Issue by kgrote** [born05/craft-twofactorauthentication#35]
It would be nice to have a config option that lets me disable the 2FA login step. This would be useful when working in development environments so I don't have to manually disable the plugin every time I sync a database from production.

**Comment by roelvanhintum** [comment]
That would be a good feature. Maybe something similar to `verifyFrontEnd`. What about `verifyBackEnd` and let it default to true?

**Comment by kgrote** [comment]
Sounds good to me!

**Comment by roelvanhintum** [comment]
@kgrote added in 2.2.0

**Comment by kgrote** [comment]
This isn't working for me. My local environment still asks for verification after logging in to the control panel. Here's my config file:

```
return [
    '*' => [  
        // Allow a totp delay in seconds (gives the user some extra time after code expired)
        'totpDelay' => null,

        'verifyFrontEnd' => false,
        'forceFrontEnd' => false,
        'forceBackEnd' => true,
        
        // The URI we should use for 2FA ligin verification on the front-end.
        'verifyPath' => '',
        
        // The URI we should use for 2FA settings (turning it on and off) on the front-end.
        'settingsPath' => '',

        // Choose between using the whitelist or blacklist! Using both will block everything!
        // Whitelist paths that do not need 2FA. Exact path or regex.
        'frontEndPathWhitelist' => [
            '*' => [],
        ],
        // Blacklist paths that do need 2FA. Exact path or regex.
        'frontEndPathBlacklist' => [
            '*' => [],
        ],
    ],
    'local' => [
        'verifyBackEnd' => false,
        'forceBackEnd'  => false
    ]
];

```

**Comment by kgrote** [comment]
Just wanted to bump this and say this config option still isn't working.

**Comment by roelvanhintum** [comment]
@kgrote sorry, completely missed this. I'll look into this today.

**Comment by roelvanhintum** [comment]
@kgrote just did some tests, config works but `userLoginEventHandler` is ignoring it. Will be fixed in the next release.

**Comment by roelvanhintum** [comment]
Fixed in 2.4.0. @kgrote can you verify this?

**Comment by kgrote** [comment]
It works, thanks!
